velox/vector/BaseVector.cpp

```diff
@@ -551,8 +551,7 @@ void BaseVector::ensureWritable(
     copy =
         BaseVector::create(isUnknownType ? type : resultType, targetSize, pool);
   }
-  SelectivityVector copyRows(
-      std::min<vector_size_t>(targetSize, result->size()));
+  SelectivityVector copyRows(result->size());
```

Comment:
This is because copyRows will always be result->size() as rows.end() cant be bigger than result->size() and targetSize is max of rows.end() and result->size().